.changeset/many-suns-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@livekit/rtc-node": patch
```

Comment:
should be minor

Reply:
hm, why? 
we're still in 0.x so minor versions are more likely to introduce breaking changes, which this doesn't

Reply:
this is a new API